### PR TITLE
Adds Ubuntu 20.04 to install instructions

### PR DESCRIPTION
**Pull request by marshallford**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

Adds Ubuntu 20.04 to install instructions

#### Which issue(s) this PR fixes:

Related to #3629

#### Special notes for your reviewer:

N/A

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by haircommander** [comment]
/ok-to-test
/approve
/lgtm

thanks @marshallford !

File: README.md [hunk 1]
```diff
@@ -158,7 +158,7 @@ wget -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:st
 echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/Raspbian_10/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
 wget -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/Raspbian_10/Release.key -O- | sudo apt-key add -
 
-# Ubuntu (18.04, 19.04 and 19.10)
+# Ubuntu (18.04, 19.04, 19.10, and 20.04)
```
Comment (haircommander) [inline README.md:161]: +1 to the oxford comma :sunglasses:

**Comment by marshallford** [comment]
CI not behaving?

**Comment by haircommander** [comment]
is it ever?

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by mrunalp** [comment]
Merging this but we probably need an issue for the failing runtime class test.

**Pull request merged**